.github/workflows/node-ci.yml

```diff
@@ -185,7 +185,7 @@ jobs:
         with:
           path: |
             .sewing-kit/
-            ./packages/*/build/
+            ./packages/*/build1/
```

Comment:
mental note to switch this back when I figure out how to clear cache properly

https://github.com/actions/cache/issues/2#issuecomment-673493515